File: code-coaches/easy/hovercraft/Hovercraft.java
import java.util.Scanner;

public class Hovercraft {

	public static String hovercraftProfitStatus(int unitsSold) {
	    int sellPrice = 3000000;
		int costPrice = 2000000;
		int insurancePrice = 1000000;

		int unitsBuilt = 10;
		int expenses = (unitsBuilt * costPrice) + insurancePrice;
		int revenue = unitsSold * sellPrice;

		if (revenue > expenses)
			return "Profit";
		else if (revenue < expenses)
			return "Loss";
		else
			return "Broke Even";
	}

	public static void main(String[] args) {
		Scanner input = new Scanner(System.in);
		int unitsSold = input.nextInt();

		System.out.println(hovercraftProfitStatus(unitsSold));

		input.close();
	}

}

File: code-coaches/easy/hovercraft/hovercraft.py
def hovercraft_profit_status(units_sold: int) -> str:
    sell_price = 3_000_000
    cost_price = 2_000_000
    insurance_price = 1_000_000

    units_built = 10
    expenses = (units_built * cost_price) + insurance_price
    revenue = units_sold * sell_price

    if revenue > expenses:
        return "Profit"
    elif revenue < expenses:
        return "Loss"
    else:
        return "Broke Even"


def main():
    units_sold = int(input())
    print(hovercraft_profit_status(units_sold))


if __name__ == "__main__":
    main()
